File: Ginger/src/net/pic4pic/ginger/entities/Gender.java
package net.pic4pic.ginger.entities;

public enum Gender implements IntegerEnum {
	
	Unknown(0),
	Male(1),
	Female(2);
	
	private final int value;
	
	private Gender(int value) {
		this.value = value;
	}

	public int getIntValue() {
		return this.value;
	}
}

File: Pic4Pic/Pic4Pic.ObjectModel/Entities/Gender.cs
using System;
using System.Runtime.Serialization;

namespace Pic4Pic.ObjectModel
{
    [DataContract()]
    public enum Gender
    {
        [EnumMember]
        Unknown = 0,

        [EnumMember]
        Male,

        [EnumMember]
        Female
    }

    public class GenderHelper
    {
        public static Gender GetGenderFromString(string gender)
        {
            if (!String.IsNullOrWhiteSpace(gender))
            {
                gender = gender.Trim();

                if (0 == String.Compare(gender, "Male", StringComparison.InvariantCultureIgnoreCase))
                {
                    return Gender.Male;
                }
                else if (0 == String.Compare(gender, "Female", StringComparison.InvariantCultureIgnoreCase))
                {
                    return Gender.Female;
                }
            }

            return Gender.Unknown;
        }
    }
}
